### DISCUSSION TITLE
Custom Icon for GeoJSON

### DISCUSSION THREAD

**Discussion by anujdivesh** [TerriaJS/terriajs#6192] [Q&A] [answered]
Hi:
Was wondering if there is a way to dynamically resize custom icon based on screen size, e.g. using width/height as percentage, etc

Sample Code:
```{
   "name":"My layer",
   "type":"geojson",
   "url":"datapoints.geojson",
   "style":{
      "marker-url":"icon.png",
      "marker-width":30,
      "marker-height":30
   },
   "description":"words here"
}
```
I use Terria v8.

**Comment by nf-s** [comment, marked as answer]
I don't believe so, the `marker-width` and `marker-height` properties must be in pixels

https://cesium.com/learn/cesiumjs/ref-doc/BillboardGraphics.html?classFilter=billboard#height

https://github.com/CesiumGS/cesium/blob/999ee4e6b82438996c30955cc47103bcfce4a35f/Source/DataSources/BillboardGraphics.js#L24-L25
